packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -87,6 +87,11 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
     // "Uninitialized" state when they are disputed on the OptimisticOracle.
     mapping(bytes32 => RelayData) public relays;
 
+    // Associates a relay request's ancillary data with the deposit hash that the relay request was linked with. This
+    // mapping is used by the OptimisticOracle callback functions (i.e. priceDisputed, priceSettled) to identify the
+    // relay request that was disputed or settled.
+    mapping(bytes => bytes32) public relayRequestAncillaryData;
```

Comment:
would changing this to a `bytes32 => bytes32` save enough gas to justify it? you could store a hash of the ancillary perhaps? wdyt?

Reply:
Yep this saves a bit of gas (<1000) but I'll take it!